Gopkg.lock

```diff
@@ -205,15 +205,13 @@
     "starlingx/inventory/v1/ptp",
     "starlingx/inventory/v1/routes",
     "starlingx/inventory/v1/serviceparameters",
-    "starlingx/inventory/v1/snmpCommunity",
-    "starlingx/inventory/v1/snmpTrapDest",
     "starlingx/inventory/v1/storagebackends",
     "starlingx/inventory/v1/storagetiers",
     "starlingx/inventory/v1/system",
     "starlingx/inventory/v1/volumegroups",
   ]
   pruneopts = "T"
-  revision = "5ecbca2671f0b94b96f9658ed5d91174b3506530"
+  revision = "05fa47c20ce5b3bd73daaf538555d4db2c3ef887"
```

Comment:
Please rebase your forked gophercloud repo and update this PR with another dep ensure -update.

Reply:
Done